yaksh/python_assertion_evaluator.py

```diff
@@ -70,23 +71,30 @@ def check_code(self):
             tb = None
             _tests = compile(self.test_case, '<string>', mode='exec')
             exec(_tests, self.exec_scope)
-        except AssertionError:
+        except TimeoutException:
+            raise
+        except Exception:
             type, value, tb = sys.exc_info()
             info = traceback.extract_tb(tb)
             fname, lineno, func, text = info[-1]
             text = str(self.test_case)
+
+            # Get truncated traceback
+            err_tb_lines = traceback.format_exc().splitlines()
+            stripped_tb_lines = []
+            for line in err_tb_lines:
+                if '.py' not in line:
```

Comment:
Instead, replace things like `File "/tmp/t.py", line 2, in foo` with a suitable change to the code if you really want to suppress the file name.  Do you have a specific case in mind?  You can use a regex for `File ".*\.py"` and replace it to what you want.  For example:
```
re.sub(r'File\s+".*?",\s+line', 'File filename, line', "File "/tmp/t.py", line 8, in <module>")
```